CONTRIBUTING.md

```diff
@@ -7,16 +7,36 @@ You will need Ruby installed, for version see: `.ruby_version` in the root.
 You will need bundler (`gem install bundler`):
 
 ```
-$ cd cookbooks-clone && bundle install
```

Comment:
@fh Sorry, I forgot to submit my comment here — but shouldn't it be `bundle exec appraisal install`?